proto/gaia/globalfee/v1beta1/tx.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package gaia.globalfee.v1beta1;
+
+option go_package = "github.com/cosmos/gaia/x/globalfee/types";
+
+import "cosmos/msg/v1/msg.proto";
+import "gaia/globalfee/v1beta1/genesis.proto";
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+
+// Msg defines the x/mint Msg service.
```

Comment:
Shouldn't it be x/globalfee?

Reply:
https://github.com/CosmosContracts/juno/pull/708/commits/cf0f3c0d681d9874ecf2d9190102a776206beb91